src/cmd/binary.js

```diff
@@ -85,21 +86,23 @@ class BinaryCommand {
 		return fileInfo;
 	}
 
-	async _verifyBundle(binaryFileInfo, assets) {
-		if (binaryFileInfo.assets && assets.length){
+	async _verifyBundle(appInfo, assets) {
+		const appAssets = appInfo.assets;
+		if (appAssets && assets.length > 0) {
 			console.log('It depends on assets:');
-			for (const assetInfo of binaryFileInfo.assets) {
-				const asset = assets.find((asset) => asset.name === assetInfo.name);
+			for (const appAsset of appAssets) {
+				const asset = assets.find((bundleAsset) => bundleAsset.name === appAsset.name);
 				if (asset) {
-					const valid = isAssetValid(asset.data, assetInfo);
+					const valid = isAssetValid(asset.data, appAsset);
+					// isAssetValid();
 
 					if (valid) {
-						console.log(' ' + chalk.bold(assetInfo.name) + ' (hash ' + assetInfo.hash + ')');
+						console.log(' ' + chalk.bold(appAsset.name) + ' (hash ' + appAsset.hash + ')');
 					} else {
-						console.log(chalk.red(' ' + assetInfo.name + ' failed' + ' (hash should be ' + assetInfo.hash + ')'));
+						console.log(chalk.red(' ' + appAsset.name + ' failed' + ' (hash should be ' + appAsset.hash + ')'));
 					}
 				} else {
-					console.log(chalk.red(' ' + assetInfo.name + ' failed' + ' (hash should be ' + assetInfo.hash + ' but is not in the bundle)'));
+					console.log(chalk.red(' ' + appAsset.name + ' failed' + ' (hash should be ' + appAsset.hash + ' but is not in the bundle)'));
 				}
```

Comment:
No functional change here. Felt like the readability of this code could be improved.